### DISCUSSION TITLE
Source code artifact logstash-logback-encoder-8.0-sources.jar in maven central spoiled?

### DISCUSSION THREAD

**Discussion by marc-** [logfellow/logstash-logback-encoder#1052] [Q&A] [answered]
Content of the archive in maven central doesn't match to the project (see screenshot). Hashes matches though. Wrong project was uploaded under `net.logstash.logback:logstash-logback-encoder:8.0` (https://repo.maven.apache.org/maven2/net/logstash/logback/logstash-logback-encoder/8.0/)?
![image](https://github.com/user-attachments/assets/af25643f-f134-421b-815d-83fff41099d9)

**Comment by marc-** [comment]
It appears that only repackaged lmax-disruptor is in the archive, no `logstash-logback-encoder` soucrce files.

**Reply by philsttr** [reply, marked as answer]
Reported in #1036, which has been fixed in 8.1
